eos/b-decays/b-to-dstar-l-nu.cc

```diff
@@ -40,35 +205,34 @@ namespace eos
 
         SwitchOption opt_q;
 
-        UsedParameter m_B;
-        
-        UsedParameter mu;
+        UsedParameter hbar;
 
         UsedParameter tau_B;
 
-        UsedParameter m_Dstar;
+        UsedParameter g_fermi;
 
         SwitchOption opt_l;
 
         UsedParameter m_l;
 
-        UsedParameter g_fermi;
+        UsedParameter m_B;
 
-        UsedParameter hbar;
+        UsedParameter m_Dstar;
+
+        UsedParameter mu;
 
         Implementation(const Parameters & p, const Options & o, ParameterUser & u) :
-        model(Model::make(o.get("model", "SM"), p, o)),
-        parameters(p),
-        opt_q(o, "q", { "u", "d" }, "d"),
-        m_B(p["mass::B_" + opt_q.value()], u),
-        // mu is the renormalization scale
-        mu(p["mu"], u),
-        tau_B(p["life_time::B_" + opt_q.value()], u),
-        m_Dstar(p["mass::D^*_" + opt_q.value()], u),
-        opt_l(o, "l", {"e", "mu", "tau"}, "mu"),
-        m_l(p["mass::" + opt_l.value()], u),
-        g_fermi(p["G_Fermi"], u),
-        hbar(p["hbar"], u)
+            model(Model::make(o.get("model", "SM"), p, o)),
+            parameters(p),
+            opt_q(o, "q", { "u", "d" }, "d"),
+            hbar(p["hbar"], u),
+            tau_B(p["life_time::B_" + opt_q.value()], u),
+            g_fermi(p["G_Fermi"], u),
+            opt_l(o, "l", {"e", "mu", "tau"}, "mu"),
+            m_l(p["mass::" + opt_l.value()], u),
+            m_B(p["mass::B_" + opt_q.value()], u),
+            m_Dstar(p["mass::D^*_" + opt_q.value()], u),
```

Comment:
I think this is wrong here, because if
 
  barB^0 -> D^+ l^- \bar{v}_l
  B^- -> D^0 l^- \bar{v}_l

You can not use the same opt_q value for B and D !?

Reply:
Christoph and I just discussed this. Using the same option is indeed correct, and Christoph's comment was in error.

Reply:
okay, as i thought.